harmonica/forward/point_mass.py

```diff
@@ -231,30 +240,61 @@ def kernel_potential_cartesian(
     easting, northing, upward, easting_p, northing_p, upward_p
 ):
     """
-    Kernel function for potential gravity field in Cartesian coordinates
+    Kernel function for potential gravitational field in Cartesian coordinates
     """
-    return 1 / distance_spherical(
-        (easting, northing, upward), (easting_p, northing_p, upward_p)
+    distance = distance_cartesian(
```

Comment:
Thanks for catching this! This was an awful mistake! :man_facepalming: 
When I see this kind of mistakes I remind myself the importance of reviewing each other's code.

Reply:
:+1: for this! We should think about why the tests didn't catch it as well.

Reply:
You are right @leouieda. Tests weren't failing because we only test potential symmetry and not the true value of the potential on a certain point. The potential holds the same symmetry both in Cartesian and spherical coordinates: two observation points located along the longitudinal axis at the same distance of the point mass measure the same gravity potential, and the same happens with two observation points located along the easting axis at the same distance of the point mass.